modin/pandas/test/test_rolling.py

```diff
@@ -91,7 +92,10 @@ def test_dataframe(data, window, min_periods, win_type, axis):
         df_equals(modin_rolled.max(), pandas_rolled.max())
         df_equals(modin_rolled.skew(), pandas_rolled.skew())
         df_equals(modin_rolled.kurt(), pandas_rolled.kurt())
-        df_equals(modin_rolled.apply(np.sum), pandas_rolled.apply(np.sum))
+        if PandasCompatVersion.CURRENT != PandasCompatVersion.PY36:
+            # skip this on pandas 1.1 as pandas raise an exception in there
```

Comment:
Don't we want to file an issue for this?

Reply:
https://github.com/modin-project/modin/pull/4800#discussion_r944652994